content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx

```diff
@@ -69,6 +69,10 @@ To include [unsupported database features](include-unsupported-database-features
    ```
 
    - If the changes are significant, it can be easier to replace the entire migration file with the result of a database dump ([`mysqldump`](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html), [`pg_dump`](https://www.postgresql.org/docs/12/app-pgdump.html))
+   
+<Admonition type="info">
+Note that the order of the tables matters when creating all of them at once, since foreign keys are created at the same step. Therefore, either re-order them or move constraint creation to the last step after all tables are created, so you won't face `can't create constraint` errors
+</Admonition>
```

Comment:
The admonition should be intended so it is rendered _inside_ the list item it clarifies.